res/values-ua/strings.xml

```diff
@@ -0,0 +1,114 @@
+	<string name="add_word_success">\"%1$s\" додано.</string>
+	<string name="add_word_title">Додати слово</string>
+
+	<string name="pref_category_about">Про програму</string>
+	<string name="pref_category_abc_mode">Режим АБВ</string>
+	<string name="pref_category_hacks">Сумістність</string>
+	<string name="pref_category_appearance">Зовнішній вигляд</string>
+	<string name="pref_category_debug_options" translatable="false">Debug Options</string>
+	<string name="pref_category_predictive_mode">Предективний режим</string>
+	<string name="pref_category_function_keys">Гарячі клавіші</string>
+	<string name="pref_category_keypad">Клавіатура</string>
+	<string name="pref_category_setup">Початкові налаштування</string>
+	<string name="pref_category_usage_stats" translatable="false">Usage Stats</string>
+
+	<string name="pref_abc_auto_accept">Автоматичний вибір літери</string>
+	<string name="pref_abc_auto_accept_summary">Автоматично вводить вибрану літеру після короткої затримки.</string>
+	<string name="pref_alternative_suggestion_scrolling">Альтернативний метод прокрутки пропозицій</string>
+	<string name="pref_alternative_suggestion_scrolling_summary">Увімкніть, якщо ви іноді не бачите всіх пропозицій або не можете їх прокрутити (Android 9 або старіші).</string>
+	<string name="pref_auto_space">Автоматичний пробіл</string>
+	<string name="pref_auto_space_summary">Автоматично додавати пробіл після розділових знаків або слів.</string>
+	<string name="pref_auto_text_case">Автоматична велика літера</string>
+	<string name="pref_auto_text_case_summary">Автоматично починати речення з великої букви</string>
+	<string name="pref_choose_languages">Мови</string>
+	<string name="pref_dark_theme">Темна тема</string>
+	<string name="pref_dark_theme_yes">Так</string>
+	<string name="pref_dark_theme_no">Ні</string>
+	<string name="pref_dark_theme_auto">Автоматично</string>
+	<string name="pref_double_zero_char">Символ при подвійному натискання клавіші 0</string>
+	<string name="pref_hack_fb_messenger">Відправляти повідомлення по натиску OK в Messenger</string>
+	<string name="pref_hack_fb_messenger_summary">Дозволяє відправляти повідомлення по натиску OK в Facebook Messenger.</string>
+	<string name="pref_show_soft_function_keys">Показувати кнопки на екрані</string>
+	<string name="pref_show_soft_numpad">Показувати цифрову клавіатуру на екрані</string>
+	<string name="pref_show_soft_numpad_summary" translatable="false">(BETA)</string>
+	<string name="pref_help">Допомога</string>
+	<string name="pref_upside_down_keys">Зворотній порядок клавіш</string>
+	<string name="pref_upside_down_keys_summary">Використовуйте це налаштування, якщо у вас в першому ряді 7–8–9 замість 1–2–3.</string>
+
+	<string name="dictionary_cancel_load">Скасувати завантаження</string>
+	<string name="dictionary_load_bad_char">Завантаження не вдалося. Невірне слово \"%1$s\" у рядку %2$d мови \"%3$s\".</string>
```

Comment:
```suggestion
	<string name="dictionary_load_bad_char">Завантаження не вдалося. Невірне слово «%1$s» у рядку %2$d мови «%3$s».</string>
```

Isn't it nicer to use the Ukrainian/French quotes instead of English ones? As I understand this is the standard.

Reply:
I don't agree at this point
Usually, double quotes are nicer in the interfaces and have much more font support

But it is technically true that we use those Christmas tree quotes, so the decision is up to you